File: 2614-maximum-count-of-positive-integer-and-negative-integer/2614-maximum-count-of-positive-integer-and-negative-integer.java
class Solution {
    public int maximumCount(int[] nums) {
        int pc = 0, nc = 0;

        for (int i = 0; i < nums.length; i++) {
            if (nums[i] < 0) {
                nc++;
            } else if (nums[i] > 0) {
                pc++;
            }
        }
        int z = Math.max(pc, nc);
        return z;
    }
}

File: 2614-maximum-count-of-positive-integer-and-negative-integer/2614-maximum-count-of-positive-integer-and-negative-integer.cpp
class Solution {
public:
    int maximumCount(vector<int>& nums) {
        int pc = 0, nc = 0;

    for (auto n : nums) {
        if (n > 0) {
            pc++;
        } else if (n < 0) {
            nc++;
        }
    }

    return max(pc, nc);
    }
};